charts/victoria-metrics-single/templates/server-service-headless.yaml

```diff
@@ -1,22 +1,22 @@
-{{- if and .Values.server.enabled .Values.server.statefulSet.enabled -}}
+{{- if .Values.server.enabled -}}
```

Comment:
two services are created with default setup, could you pleas hide creation of headless by configuration?